### ISSUE TITLE
Snapshot and Pause doesn't work after upgrading GStreamer

### ISSUE THREAD

**Issue by liutingdu** [evercam/evercam-play-android#108]
I upgraded GStreamer SDK from 1.4.5 to 1.6.0, built with latest NDK(r10e) with no error.

Then I noticed that the black screen issue (https://github.com/evercam/evercam-play-android/issues/42) is not happening again with the new version of GStreamer.

But got new problems:

- Snapshot doesn't work.  
After clicking the save snapshot icon for camera 'OConnelStr', it shows error message "Requesting snapshot failed" with the following errors in logs: 

```
10-08 14:39:15.075 23418-24117/io.evercam.androidapp W/GStreamer+basetransform: 0:00:11.901803954 0x972dea30 gstbasetransform.c:1414:gst_base_transform_setcaps:<videoscale0> transform could not transform video/x-raw(memory:GLMemory, meta:GstVideoOverlayComposition), width=(int)704, height=(int)576, interlace-mode=(string)progressive, pixel-aspect-ratio=(fraction)1/1, framerate=(fraction)0/1, format=(string)RGBA in anything we support
10-08 14:39:15.076 23418-24117/io.evercam.androidapp W/GStreamer+basetransform: 0:00:11.902859058 0x972dea30 gstbasetransform.c:1414:gst_base_transform_setcaps:<videoscale0> transform could not transform video/x-raw(memory:GLMemory, meta:GstVideoOverlayComposition), width=(int)704, height=(int)576, interlace-mode=(string)progressive, pixel-aspect-ratio=(fraction)1/1, framerate=(fraction)0/1, format=(string)RGBA in anything we support
10-08 14:39:15.077 23418-24117/io.evercam.androidapp W/GStreamer+basetransform: 0:00:11.903232652 0x972dea30 gstbasetransform.c:1414:gst_base_transform_setcaps:<videoscale0> transform could not transform video/x-raw(memory:GLMemory, meta:GstVideoOverlayComposition), width=(int)704, height=(int)576, interlace-mode=(string)progressive, pixel-aspect-ratio=(fraction)1/1, framerate=(fraction)0/1, format=(string)RGBA in anything we support
10-08 14:39:15.077 23418-24117/io.evercam.androidapp W/GStreamer+basesrc: 0:00:11.903379267 0x972dea30 gstbasesrc.c:2943:gst_base_src_loop:<appsrc0> error: Internal data flow error.
10-08 14:39:15.077 23418-24117/io.evercam.androidapp W/GStreamer+basesrc: 0:00:11.903475100 0x972dea30 gstbasesrc.c:2943:gst_base_src_loop:<appsrc0> error: streaming task paused, reason not-negotiated (-4)
10-08 14:39:15.078 23418-24117/io.evercam.androidapp W/GStreamer+basetransform: 0:00:11.903956038 0x972dea30 gstbasetransform.c:1414:gst_base_transform_setcaps:<videoscale0> transform could not transform video/x-raw(memory:GLMemory, meta:GstVideoOverlayComposition), width=(int)704, height=(int)576, interlace-mode=(string)progressive, pixel-aspect-ratio=(fraction)1/1, framerate=(fraction)0/1, format=(string)RGBA in anything we support
10-08 14:39:15.078 23418-24315/io.evercam.androidapp E/GStreamer+default: 0:00:11.904187913 0x955e58e0 convertframe.c:325:gst_video_convert_sample Could not convert video frame: Internal data flow error.
10-08 14:39:15.079 23418-24117/io.evercam.androidapp W/GStreamer+basetransform: 0:00:11.905014892 0x972dea30 gstbasetransform.c:1414:gst_base_transform_setcaps:<videoscale0> transform could not transform video/x-raw(memory:GLMemory, meta:GstVideoOverlayComposition), width=(int)704, height=(int)576, interlace-mode=(string)progressive, pixel-aspect-ratio=(fraction)1/1, framerate=(fraction)0/1, format=(string)RGBA in anything we support
10-08 14:39:15.080 23418-24315/io.evercam.androidapp D/GStreamer+evercam: 0:00:11.906269944 0x955e58e0 jni/evercam.cpp:63:attach_current_thread Attaching thread 0x955e58e0
10-08 14:39:15.084 23418-24315/io.evercam.androidapp D/GStreamer+evercam: 0:00:11.910684267 0x955e58e0 jni/evercam.cpp:78:detach_current_thread Detaching thread 0x955e58e0
```

- 'Pause' button doesn't work well  
This is for camera 'OconnellSt', when I press the 'Pause' button, got the following warning in logs: 
```
10-08 14:48:07.450 23418-3244/io.evercam.androidapp W/GStreamer+rtspsrc: 0:09:04.276883438 0x92be1a00 gstrtspsrc.c:6051:gst_rtspsrc_send:<source> error: Got error response: 551 (Option not supported).
10-08 14:48:07.451 23418-3244/io.evercam.androidapp W/GStreamer+rtspsrc: 0:09:04.277192240 0x92be1a00 gstrtspsrc.c:8011:gst_rtspsrc_pause:<source> error: Could not send message. (Generic error)
```

Then I press the 'Play' button, the video is not resumed.

Tried different matches between /1.4.5/1.5.2/1.6.0 and NDK r10c/r10d/r10e, it always happening for 1.5.2 & 1.6.0 no matter which NDK version I use.

@GNUDimarik

**Comment by GNUDimarik** [comment]
It should work, please test

**Comment by liutingdu** [comment]
Fixed by using our own video sink and stop using playbin. https://github.com/evercam/evercam-play-android/pull/119 Published with app version 1.6.5
